pallets/ddc-payouts/src/lib.rs

```diff
@@ -947,6 +963,13 @@ pub mod pallet {
 				amount_to_deduct.saturated_into::<BalanceOf<T>>(),
 				ExistenceRequirement::AllowDeath,
 			)?;
+
+			pallet::Pallet::deposit_event(Event::<T>::ValidatorsRewarded {
```

Comment:
rename ValidatorsRewarded to ValidatorRewarded cause it's only 1 validator

Reply:
Updated